src/app/ui.js

```diff
@@ -49,6 +57,40 @@ class UI {
     const newTask = document.querySelector('.task-input');
     newTask.value = '';
   }
```

Comment:
Kindly fix this issue:

- Add a function for remove single `items` from the array requested in your `project requirements`:


![Screen Shot 2021-12-04 at 8 07 49](https://user-images.githubusercontent.com/22258274/144712481-9edb602b-4c1e-483a-9159-c8a36fe93143.png)


You have a `function` for removes selected items.
Try to follow all the requirements requested before to request another code review.

Reply:
![change](https://user-images.githubusercontent.com/84543481/144713148-4f4508d1-9c01-4f3f-b2f0-2b3a2672eb0f.PNG)
And this is the part for editing the task description. Please check again or shall I send a video that shows everything in action?
You already can remove selected ITEMS or SINGLE ITEM depending on your selection. That method is also deleteFromSTore().